### PR TITLE
Replace AskDarcel with SF Service Guide in Privacy/Terms

### PR DESCRIPTION
**Pull request by Raawr**
#760 1.2 Volunteer

### CODE REVIEW

File: app/pages/legal/PrivacyPolicy.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export class PrivacyPolicyPage extends React.Component {
   static get content() {
     return `
 # Privacy Policy
-We do everything within our power to protect your privacy given the sensitive nature of many of the services listed on AskDarcel.
```
Comment (jjfreund) [inline app/pages/legal/PrivacyPolicy.jsx]: It seems like changes related to your other pull request ended up here -- best to keep them separate so we can review and merge them in one at a time.

**Comment by richardxia** [comment]
Looks like this PR has two separate changes, one for #760 and one for #782. The title of this PR reflects the change necessary for #782, but that has already been done in @daniel-owens' PR https://github.com/ShelterTechSF/askdarcel-web/pull/797. I can retitle this PR to reflect only #760 and remove the commit that fixes #782 from this branch.

File: app/pages/legal/PrivacyPolicy.jsx (hunk 1 above)
Comment (Raawr) [reply app/pages/legal/PrivacyPolicy.jsx]: Ah totally that makes sense, thank you!

**Comment by Raawr** [comment]
Thank you Richard!!

**Pull request merged**